app/lib/admin/actions/moderate_package.dart

```diff
@@ -16,11 +18,19 @@ final moderatePackage = AdminAction(
 Set the moderated flag on a package (updating the flag and the timestamp).
 ''',
   options: {
+    'case': 'The ModerationCase.caseId that this action is part of.',
     'package': 'The package name to be moderated',
     'state':
         'Set moderated state true / false. Returns current state if omitted.',
+    'message': 'Optional message to store and send out.'
```

Comment:
```suggestion
    'message': 'Optional message to store.'
```

Are we sending it to anyone?

Reply:
Not yet, but I though we should if there is a message.